components/breadcrumb/stories/breadcrumb.mdx

```diff
@@ -0,0 +1,56 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as BreadcrumbStories from './breadcrumb.stories';
+
+<Meta of={BreadcrumbStories} title="Docs" />
+
+<Title of={BreadcrumbStories} />
+<Description of={BreadcrumbStories} />
+
+### Standard
+
+<Canvas of={BreadcrumbStories.Default} />
+
+### Dragged
+
+<Canvas of={BreadcrumbStories.Dragged} />
+
+### Nested
+
+<Canvas of={BreadcrumbStories.Nested} />
+
+### Nested (root visible)
+
+<Canvas of={BreadcrumbStories.NestedRootVisible} />
+
+### Multiline
+
+<Canvas of={BreadcrumbStories.Multiline} />
+
+### Multiline (dragged)
+
+<Canvas of={BreadcrumbStories.MultilineDragged} />
+
+### Multiline Nested
```

Comment:
I love this, that's a really great point about the first-time visitors, added it in here: https://github.com/adobe/spectrum-css/pull/2793/commits/8dd9fca301b60977c187de2406cd2cdd26adbd60

However, in trying to work that in, I may have found a bug for the `is-disabled` state. It seems like that class is added to the `.spectrum-Breadcrumbs-item`/parent `li`, instead of `.spectrum-Breadcrumbs-itemLink`. We had a block for disabled styles for `.spectrum-Breadcrumbs-itemLink &.is-disabled`, but that's not the behavior I was getting. [This fix](https://github.com/adobe/spectrum-css/pull/2793/commits/27b5ef2f056948b42247cdcefc6871c24f3a551e) seems to do what I'm expecting, but am I understanding things all the way? Am I missing anything? Should I be declaring `isDisabled` somewhere else in the components?

Reply:
per our slack conversation about fixing where the `is-disabled` is being applied in the template, here's the relevant commits! 

[fix the template](https://github.com/adobe/spectrum-css/pull/2793/commits/d9702da4cbbc85d053b949c261711f0aed10df06)
[add disabled story to mdx with is-disabled notes](https://github.com/adobe/spectrum-css/pull/2793/commits/a7dbbb9337f2f7e34908c0220c0375925032ff54)